README.md

```diff
@@ -237,7 +264,7 @@ gcloud run deploy "litellm-proxy" \
     --execution-environment=gen1 \
     --description="LiteLLM Proxy" \
     --region="$MY_REGION" \
-    --set-env-vars="LITELLM_MODE=PRODUCTION,LITELLM_LOG=ERROR,VERTEXAI_PROJECT=${MY_PROJECT_ID},VERTEXAI_LOCATION=${MY_REGION},LITELLM_MASTER_KEY=sk-${MY_RANDOM}" \
+    --set-env-vars="LITELLM_MODE=PRODUCTION,LITELLM_LOG=ERROR,VERTEXAI_PROJECT=${MY_PROJECT_ID},VERTEXAI_LOCATION=${MY_REGION},LITELLM_MASTER_KEY=sk-${MY_RANDOM},GOOGLE_CLOUD_PROJECT=${MY_PROJECT_ID}" \
```

Comment:
`GOOGLE_CLOUD_PROJECT` is unnecessary as the value of `VERTEXAI_PROJECT` is already the same

Reply:
Devin's conclusion that both these variables should be included actually shows some interesting foresight. If taken at face value it could indicate that there are some provided best practices for these sorts of solutions as technically the VertexAI project and the GCP project holding Arize secrets could be separated.

Deferring to Devin here!